data-pipeline/src/pipeline/mappings.py

```diff
@@ -205,40 +140,70 @@ def map_is_surplus_deficit(closing_balance: float):
         return "Deficit"
 
 
-def _diff_month(d1, d2, opening=False):
-    """
-    The number of months between two dates (exclusive).
-
-    Note: for opening-dates, where d2 is up to the 10th September, this
-    is considered a full year. Also, the opening month is considered a full
-    month whereas closing isn't (i.e. opened on 1st Oct is 11 months,
-    whereas closed on the 1 Aug is only 11 months).
-
-
-    :param d1: end date
-    :param d2: start date
-    :param opening: whether to consider this an org. "opening" date
-    :return: number of months between start and end dates
-    """
-    if opening:
-        if d2.month == 9:
-            return 12 if d2.day <= 10 else 11
-        else:
-            return (d1.year - d2.year) * 12 + d1.month - d2.month + 1
-
-    return (d1.year - d2.year) * 12 + d1.month - d2.month
+def _period_from_start(start):
```

Comment:
Nitpick: could either benefit from a docstring or perhaps just calling it `_months_from_start()` would make it clearer…?

Reply:
Actually, this one confused me: I though this was "period-from-the-start-of-the-academic-year" (but it's "period-from-this-start-date-to-the-end-of-the-academic-year").

Maybe `_months_left_in_academic_year`/`_months_spent_in_academic_year` or similar might be clearer…?

Reply:
Only think to consider, this is a specific calc for the academy return. The dates align to the AAR return period (which is the academic year), it would be a different calc for maintained schools.

Reply:
@PsypherPunk methods renamed